enginetest/enginetests.go

```diff
@@ -73,15 +73,6 @@ func TestQueries(t *testing.T, harness Harness) {
 			TestQuery2(t, harness, e, tt.Query, tt.Expected, tt.ExpectedColumns, nil)
 		}
 	}
-
-	t.Run("date parse tests", func(t *testing.T) {
-		harness.Setup(setup.MydbData)
-		for _, tt := range queries.DateParseQueries {
-			t.Run(tt.Query, func(t *testing.T) {
-				TestQueryWithEngine(t, harness, e, tt)
-			})
-		}
-	})
```

Comment:
I added this here before, but these queries are tested in `TestDateParse`, so removing it.